setup.sh

```diff
@@ -17,13 +17,14 @@ if ! id -u bb8 > /dev/null 2>&1; then
     echo "bb8:$password" | chpasswd
 
     mkdir -p /var/lib/bb8/.ssh
-
+    
     # add to docker group
     usermod -aG docker bb8
 fi
 
 # give bb8 group ownership of this dir
 chgrp -R bb8 $HERE
+chmod -R 775 $HERE
```

Comment:
Needed to make sure members of `bb8` group to write to the directory

Reply:
Is the -R needed?  That will set the executable bit on all files.  Might be better to do `chmod -R g+w $HERE` instead?  (sorry for the misdirection)

Reply:
So does `chmod -R g+w $HERE` give write permissions for the group to the whole directory and subdirectories?

Reply:
That's right - so rather than setting all permissions to `rwxrwxr-x` which is what the 775 would do, this adds the `w` bit for the group - I should have thought about this when we discussed the other day